README.md

```diff
@@ -1 +1,156 @@
-# opencraft
+OpenCraft
+=========
+
+Install
+-------
+
+Instructions based on Ubuntu 14.04.
+
+Install the system package dependencies & virtualenv:
+
+```
+$ sudo apt-get install `cat debian_packages.lst`
+$ pip3 install --user virtualenv && pip3 install --user virtualenvwrapper
+```
+
+Ensure you load virtualenv with Python 3 in `~/.bashrc`:
+
+```
+export PATH="$PATH:$HOME/.local/bin" VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+source $HOME/.local/bin/virtualenvwrapper.sh
```

Comment:
@bradenmacdonald Good point - added to the paragraph below, since it only makes sense to add it to the `.bashrc` when that's the only virtualenv the current user will use. Which isn't the case, for example, for my local dev environment (but might be on yours if you use a VM for that).